containers/py38/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# hadolint ignore=DL3006
+FROM alpine
+
+# install Python3, and uses pip to install pytest
+# finally it removes the cache - all in one go to prevent another layer which we don't need
+RUN set -ex && \
+# install python, pip,  and pytest \
+    apk add --no-cache python3=3.8.5-r0 && \
```

Comment:
I have an issue when the python version is specified it does not install python. I removed the exact version and everything works.
```
diff --git a/containers/py38/Dockerfile b/containers/py38/Dockerfile
index d3d6eb0..2e77649 100644
--- a/containers/py38/Dockerfile
+++ b/containers/py38/Dockerfile
@@ -5,7 +5,7 @@ FROM alpine
 # finally it removes the cache - all in one go to prevent another layer which we don't need
 RUN set -ex && \
 # install python, pip,  and pytest \
-    apk add --no-cache python3=3.8.5-r0 && \
+    apk add --no-cache python3 && \
     python3 -m ensurepip && \
     pip3 install pytest==6.1.1 && \
     rm -rf /var/cache/apk/* && \
```

Reply:
Python is removed at the end of this block... so it is not expected to stick around - is that what you are seeing?